demo-plugin/webpack.config.ts

```diff
@@ -0,0 +1,66 @@
+/* Copyright Contributors to the Open Cluster Management project */
+/* eslint-env node */
+
+import * as path from 'path'
+import { ConsoleRemotePlugin } from '@openshift-console/dynamic-plugin-sdk-webpack'
+import * as webpack from 'webpack'
+import type { Configuration as DevServerConfiguration } from 'webpack-dev-server'
+
+const config: webpack.Configuration & DevServerConfiguration = {
+    mode: 'development',
+    context: path.resolve(__dirname, 'src'),
+    entry: {},
+    output: {
+        path: path.resolve('./dist'),
+        filename: '[name]-bundle.js',
+        chunkFilename: '[name]-chunk.js',
+    },
+    watchOptions: {
+        ignored: ['node_modules', 'dist'],
+    },
+    devServer: {
+        port: 9001,
+        devMiddleware: {
+            writeToDisk: true,
+        },
+        static: ['dist'],
+    },
+    resolve: {
+        extensions: ['.ts', '.tsx', '.js', '.jsx'],
```

Comment:
```suggestion
        extensions: ['.ts', '.tsx'],
```
We don't need to support js or jsx file.